packages/webdriver/protocol/chromium.json

```diff
@@ -414,5 +414,36 @@
         "description": "The base64-encoded PNG image data comprising the screenshot of the visible region of an element’s bounding rectangle after it has been scrolled into view."
       }
     }
+  },
+  "/session/:sessionId/se/log/types": {
```

Comment:
```suggestion
  "/session/:sessionId/log/types": {
```

Reply:
> it should be with se in url according to issue mentioned in PR, it works with se, I have verified it

sorry I missed that

Reply:
@christian-bromann one more note here, getLogs will stop working for chrome versions (not chromedriver!) below 76.

The only way to workaround it is to check if chrome version is below 76 in runtime using `browser.capabilities.version` in jsonwp mode and `browser.capabilities.browserVersion` in w3c mode.

I don't think we really need it as far as it is better to use latest chrome, but I might be wrong.

What do you think?

Reply:
I think it is worth implementing such feature and decide on a pattern for the protocols (e.g. chromium.json is base and chromium_64.json overwrites everything from version 64 upwards). Given that we make a session request and then create the instance prototype the effort should be relatively small. That said, it should not be part of this PR. I will create an issue with this we some more details in a sec.